docs/docs/start/self-hosting.mdx

```diff
@@ -13,6 +13,8 @@ The codebase includes production-ready set of `Dockerfiles` to allow you to buil
 
 You will find these in the [infra/prod/front/Dockerfile](https://github.com/twentyhq/twenty/blob/main/infra/prod/front/Dockerfile) and [infra/prod/server/Dockerfile](https://github.com/twentyhq/twenty/blob/main/infra/prod/server/Dockerfile) files.
 
+Prebuilt images can be found on [docker hub](https://hub.docker.com/r/twentycrm/).
```

Comment:
@bkleiner I think we can remove the mention of infra/prod Dockerfiles and only talk about the dockerhub ones

Reply:
@charlesBochet i guess that's for you to decide, i just discovered the prebuilts after looking around the issues and figured i would try to spare others from searching.

ultimately something like docker compose file with the prebuilts would be appropriate imho. thats usually a good option to get one up and running quickly.

Reply:
@bkleiner could you update the documentation to remove the two lines above?
Also, would love to add a docker-compose using the dev images if you want to work on it, it's a good idea. Otherwise, we will work on it later :)

Reply:
noticed somebody else just put up a page explaining the env vars.
rebased and added a short blurb about that.